src/users/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""'users' package's initialization module."""
+
+from . import routes  # noqa
```

Comment:
Do we really need this import here? I'm not sure that its used somehow. Seems like everything will work fine without it.

Reply:
After a little research I figured out, that we don't need to change url routes, because path in this `url_for` function is defined by the first argument of the blueprint, and not by the package layout.

It means that if we have such blueprint:
```python
bp = Blueprint("users", __name__)
```
then all routes registered to it will have path `"users.route_name"`. And it doesn't depend of the package where this blueprint is defined.